crates/orchestrator/src/tests/jobs/da_job/mod.rs

```diff
@@ -91,7 +92,7 @@ async fn test_process_job() {
             )
             .await
             .unwrap(),
-        internal_id
+        "da_job_process_done"
```

Comment:
it should be the external id returned by the Da client. (txn_hash for alt_da, maybe "NA" for eth)

Reply:
returning `0xbeef` for eth